### PR TITLE
[Docs] Add demo.md Chinese Version

### PR DESCRIPTION
**Pull request by DCNSW**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Add demo.md Chinese version.

## Modification
Add demo.md Chinese version.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
no.

## Use cases (Optional)
no.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs_zh-CN/demo.md [hunk 1]
```diff
@@ -1 +1,85 @@
-# 演示
\ No newline at end of file
+# 演示
+
+我们提供了多模态/单模态（基于激光雷达/视觉）、室内/室外场景的3D检测和3D语义分割演示的脚本，预训练模型可以从 [model zoo](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/model_zoo.md/) 下载。我们也提供了 KITTI、SUN RGB-D、nuScenes 和 ScanNet 数据集的预处理样本数据，你可以根据我们的预处理步骤使用任何其它数据。
```
Comment (Wuziyi616) [inline docs_zh-CN/demo.md:3]: Maybe add space between English word `3D` and the surrounding Chinese words?

Comment (DCNSW) [reply docs_zh-CN/demo.md]: Problems solved now.

File: docs_zh-CN/demo.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+python demo/pcd_demo.py demo/data/kitti/kitti_000008.bin configs/second/hv_second_secfpn_6x8_80e_kitti-3d-car.py checkpoints/hv_second_secfpn_6x8_80e_kitti-3d-car_20200620_230238-393f000c.pth
+```
+
+在 SUN RGB-D 数据上测试 [VoteNet](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/votenet) 模型：
+
+```shell
+python demo/pcd_demo.py demo/data/sunrgbd/sunrgbd_000017.bin configs/votenet/votenet_16x8_sunrgbd-3d-10class.py checkpoints/votenet_16x8_sunrgbd-3d-10class_20200620_230238-4483c0c0.pth
+```
+
+如果你正在使用的 mmdetection3d 版本 >= 0.6.0，记住转换 VoteNet 的模型数据，查看 [README](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/votenet/README.md/) 来获取转换模型数据的详细说明。
```
Comment (Wuziyi616) [inline docs_zh-CN/demo.md:31]: 这里应该是转换 模型权重 而不是 模型数据？

**Review by Wuziyi616 — APPROVED** [review]
没什么大问题，修了这个 typo 就差不多

File: docs_zh-CN/demo.md [hunk 3]
```diff
@@ -1 +1,85 @@
...
+
+### 3D 分割
+
+在点云数据上测试 3D 分割器，运行：
+
+```shell
+python demo/pc_seg_demo.py ${PCD_FILE} ${CONFIG_FILE} ${CHECKPOINT_FILE} [--device ${GPU_ID}] [--out-dir ${OUT_DIR}] [--show]
+```
+
+可视化结果会被保存在 `${OUT_DIR}/PCD_NAME`，其中包括点云以及预测的 3D 分割掩膜。
```
Comment (Wuziyi616) [inline docs_zh-CN/demo.md:79]: 分割掩膜 建议改成 分割掩码？感觉 mask 翻译成掩码会更多一些（知乎上的经验hhh）

Comment (DCNSW) [reply docs_zh-CN/demo.md]: 已修

File: docs_zh-CN/demo.md [hunk 4]
```diff
@@ -1 +1,87 @@
-# 演示
\ No newline at end of file
+# 样例
+
+# 介绍
+
+我们提供了多模态/单模态（基于激光雷达/视觉）、室内/室外场景的 3D 检测和 3D 语义分割样例的脚本，预训练模型可以从 [Model Zoo](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/model_zoo.md/) 下载。我们也提供了 KITTI、SUN RGB-D、nuScenes 和 ScanNet 数据集的预处理样本数据，你可以根据我们的预处理步骤使用任何其它数据。
```
Comment (ZwwWayne) [inline docs_zh-CN/demo.md:5]: 视觉 -> 图像

Comment (DCNSW) [reply docs_zh-CN/demo.md]: 已修正

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**